jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,146 @@
+#!groovy
+// by default build number is used as cluster to prevent clashes with possibly already existing clusters
+String clusterName = BUILD_NUMBER
+String awsRegion = params.AWS_REGION
+String baseDomain = "intlyqe.com"
+if (CLUSTER_NAME) {
+    clusterName = CLUSTER_NAME
+}
+String clusterInfoDirectory = "clusters/" + clusterName
+String installConfigDirectory = ""
+String rootWorkspaceDir = ""
+
+def err = null
+String adminPassword = ""
+
+node("cirhos_rhel7") {
+    timeout(120) { ansiColor("gnome-terminal") { timestamps {
+        try {
+            currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
+            currentBuild.description = "clusterName: ${clusterName}\nawsRegion: ${awsRegion}"
+            
+            stage("Configure aws cli") {
+                withCredentials([string(credentialsId: "qe-iam-access-key-id", variable: "AWS_ACCESS_KEY_ID")]) {
+                    withCredentials([string(credentialsId: "qe-iam-access-secret-key", variable: "AWS_SECRET_ACCESS_KEY")]) {                
+                        sh(
+                            returnStdout: false,
+                            script: """
+                                rm -rf ~/.aws
+                                mkdir ~/.aws
+                                touch ~/.aws/credentials
+                                # set +x # suppress output of the next shell command
+                                echo "[default]\naws_access_key_id=${AWS_ACCESS_KEY_ID}\naws_secret_access_key=${AWS_SECRET_ACCESS_KEY}" > ~/.aws/credentials
+                                # set -x
+                            """
+                        )
+                    }
+                }
+            } // stage
+            
+            stage("Prepare the repo to store created cluster information") {
+                dir("rhmi-jenkins") {
+                    git branch: "master", url: "https://gitlab.cee.redhat.com/integreatly-qe/rhmi-jenkins.git"
+                    def dirExists = fileExists(clusterInfoDirectory)
+                    if (dirExists) {
+                        sh "rm -rf ${clusterInfoDirectory}"
+                    } 
```

Comment:
Should we be deleting this? Seems like if there is already a cluster directory there it would mean there is already a cluster with that name, and it should just fail with an error?

Reply:
Good point. I will change it

Reply:
:+1: